.github/workflows/run_tests.yaml

```diff
@@ -12,6 +12,7 @@ on:
   push:
     branches:
       - master
+      - mb/879-retry
```

Comment:
```suggestion
```

Can we delete this?

Reply:
Ahh woops good catch - I wanted to re-run the actions without having to make a PR whilte testing - thanks!